mlir/include/mlir/Dialect/Rock/Passes.td

```diff
@@ -127,4 +127,9 @@ def RockCheckResidencyPass : Pass<"rock-check-residency"> {
   let dependentDialects = ["LLVM::LLVMDialect"];
 }
 
+def RockGemmOutputSwizzlePass : Pass<"rock-gemm-output-swizzle", "::mlir::func::FuncOp"> {
+  let summary = "This pass changes ThreadwiseWriteAllOp to swizzle on LDS before writing to GPU memory";
+  let dependentDialects = ["rock::RockDialect", "affine::AffineDialect", "gpu::GPUDialect", "vector::VectorDialect", "memref::MemRefDialect", "linalg::LinalgDialect", "scf::SCFDialect"];
```

Comment:
You only need to depend on a dialect if you're referring to it in the pass and can't rely on it already being present. So you'll want `memref::MemRefDialect` and `rock::RockDialect` but you don't need `SCF`, say. (Heck, check your `#include`s to see if you're missing any)